### ISSUE TITLE
Add back 1.12.2 support for MMOItems and MMOCore

### ISSUE THREAD

**Issue by MateoNikolic** [mmopluginteam/mmoitems#111]
Add back 1.12.2 support for latest releases.
It's very important for us who have large servers, we have custom coded plugins specifically made for 1.12.2 and it's hard to update on newer version. Even tho I do plan to upgrade when 1.16 comes out

**Comment by Flirtey** [comment]
@MateoNikolic With how much extra work it takes to support 1.12.2, and the amount of features that we are held back on supporting 1.12.2, it's not worth it.

Server performance wise, 1.15.2 is not as far behind 1.12.2 as people think. As someone who runs several servers that held a few hundred players concurrently on 1.15.2, I know it is fully possible to hold players on newer versions.

Refusing to update your custom coded plugins and instead ask public plugins to continue supporting old versions is something known as Technological Debt and is only further delaying the problem and making it worse, by pushing your issues elsewhere.

**Comment by MateoNikolic** [comment]
I do thing that at least some version that are bug free should be supported by 1.12.2.
Having more than 500 players at the time on my server I already experimented with newer versions and they all ended up in failure even though I've configured them to perfection.
1.12.2 is the latest best optimized version for Minecraft, even 1.16 is promising and they indeed promise some amazing optimizations I do thing waiting for 1.16 is the best choice not rush anything.

Also keep in my mind I don't have a network, I have a server. It's a skyblock server and I don't have multiple servers. Maybe if I had multiple servers it'd be a different story but the way it is now, there is not way in hell that 1.15 would work as good as 1.12 at least in my situation.

This isn't by any way Technological Debt because you aren't loosing anything by keeping 1.12 support. 10% of all servers are still on 1.12 and you also have to keep in mind that people are creating test servers on latest version all the time, so that information isn't 100% to be taken seriously. I'd estimate from personal experience and from experience with other people I've talk to and I've talk to A LOT server owners. There's definitely over 80% servers that are still on 1.8.8- 1.12.2 and those are SERIOUS servers, I'm not talking about little vanilla test servers.

**Comment by MateoNikolic** [comment]
> 
> 
> @MateoNikolic With how much extra work it takes to support 1.12.2, and the amount of features that we are held back on supporting 1.12.2, it's not worth it.
> 
> Server performance wise, 1.15.2 is not as far behind 1.12.2 as people think. As someone who runs several servers that held a few hundred players concurrently on 1.15.2, I know it is fully possible to hold players on newer versions.
> 
> Refusing to update your custom coded plugins and instead ask public plugins to continue supporting old versions is something known as Technological Debt and is only further delaying the problem and making it worse, by pushing your issues elsewhere.

I do thing that at least some version that are bug free should be supported by 1.12.2.
Having more than 500 players at the time on my server I already experimented with newer versions and they all ended up in failure even though I've configured them to perfection.
1.12.2 is the latest best optimized version for Minecraft, even 1.16 is promising and they indeed promise some amazing optimizations I do thing waiting for 1.16 is the best choice not rush anything.

Also keep in my mind I don't have a network, I have a server. It's a skyblock server and I don't have multiple servers. Maybe if I had multiple servers it'd be a different story but the way it is now, there is not way in hell that 1.15 would work as good as 1.12 at least in my situation.

This isn't by any way Technological Debt because you aren't loosing anything by keeping 1.12 support. 10% of all servers are still on 1.12 and you also have to keep in mind that people are creating test servers on latest version all the time, so that information isn't 100% to be taken seriously. I'd estimate from personal experience and from experience with other people I've talk to and I've talk to A LOT server owners. There's definitely over 80% servers that are still on 1.8.8- 1.12.2 and those are SERIOUS servers, I'm not talking about little vanilla test servers.
